common/widgets/misc.module.ts

```diff
@@ -0,0 +1,16 @@
+import {BrowserModule} from '@angular/platform-browser';
+import {CommonModule} from '@angular/common';
+import {HsMiscRecursiveDd} from './recursive-dd.component';
+import {NgModule} from '@angular/core';
+/**
+ * @namespace HsMiscModule
+ * @memberOf hs
+ */
+@NgModule({
+  declarations: [HsMiscRecursiveDd],
+  imports: [CommonModule, BrowserModule],
+  providers: [],
+  entryComponents: [HsMiscRecursiveDd],
+  exports: [HsMiscRecursiveDd],
+})
+export class HsMiscModule {}
```

Comment:
I believe a better name can be invented for this module+component+service. But I can't found any appropriate now..

Reply:
Great, I like this!